mongodb/driver/query.xml

```diff
@@ -94,9 +94,9 @@
   <section xml:id="mongodb-driver-query.constants">
    &reftitle.constants;
    <para>
-    These flags correspond to the MongoDB wire protocol. They may be bitwise
-    or'd together. They may modify how a query is performed in the MongoDB
-    server.
+    These bit flags correspond to the MongoDB wire protocol for an OP_QUERY
```

Comment:
The fact the opcode is called OP_QUERY in the documentations of mongodb is irrelevant